test_api_performance.py

```
@@ -0,0 +1,45 @@
+import time
+
+# 测试项目列表API
+def test_projects_api():
+    print("Testing /projects API...")
+
+    # 记录开始时间
+    start_time = time.time()
+
+    try:
+        # 调用项目列表API
+        response = requests.get("http://localhost:8000/projects?page=1&per_page=5")
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)